litt/src/main.rs

```diff
@@ -116,25 +118,23 @@ fn main() -> Result<(), LittError> {
                 .get(last_result)
                 .expect("Number not in last results");
             if path.0.ends_with("pdf") {
-                open_pdf(path.0.clone(), path.1, path.2.clone())?;
+                open_pdf(path.0.clone(), path.1, path.2.clone(), &cli)?;
             } else {
                 open_std_programm(path.0.clone())?;
             }
             return Ok(());
         }
     }
 
-    let cli = Cli::parse();
-
     // everything that does not require litt index
 
     // Print existing litt indices
     if cli.list {
-        println!("Currently available indices:");
+        cli.display(Message::Info("Currently available indices:"));
```

Comment:
Got it. I thought it might be good to emit semantic messages – so that the message knows what content it actually contains and then the implementation to display it, decides how to display that content. What do you think about that? See latest commits for this :)